src/test/java/org/cyclonedx/BomJsonGeneratorTest.java

```diff
@@ -46,6 +46,17 @@ public void schema12GenerationTest() throws Exception {
         Assert.assertTrue(parser.isValid(file, CycloneDxSchema.Version.VERSION_12, true));
     }
 
+    @Test
+    public void schema12GenerationTestWithVulns() throws Exception {
+        Bom bom = createCommonBom("/bom-1.2-vulnerability.xml");
+        BomJsonGenerator generator = BomGeneratorFactory.createJson(Version.VERSION_12, bom);
+        Assert.assertTrue(generator instanceof BomJsonGenerator12);
+        Assert.assertEquals(CycloneDxSchema.Version.VERSION_12, generator.getSchemaVersion());
+        File file = writeToFile(generator.toJsonString());
+        JsonParser parser = new JsonParser();
+        Assert.assertTrue(parser.isValid(file, CycloneDxSchema.Version.VERSION_12, true));
```

Comment:
Failing atm, we need to register a vulns schema or something akin so it knows it's good to go